cubes/new_cube.py

```diff
@@ -240,7 +248,8 @@ def main(Cube):
 main(Cube)
 
 '''
-python new_cube.py -n 1 -m 3 -p all_params.json -o tmp.npy --clobber
+python new_cube.py -n 1 -m 3 -p all_params.json -o tmp.out --clobber
+python new_cube.py -n 1 -m 3 -p H0_params.json -o H0.out --clobber
 starti is  0
 cosmology: CosmoParams(H0=67.66, Omega_k=0.0, Omega_lambda=0.6888463055445441, Omega_m=0.30966, Omega_b=0.04897, Omega_b_h2=0.0224178568132, fixed_H0=67.66, fix_Omega_b_h2=True)
 FIX THIS!!!!!
```

Comment:
What exactly needs to be fixed? they need to be updated? provided those .json files are easy to read/write by a human this seems fine. Do they need a special method to read them?

Reply:
Also: what do n and m do? Is the idea that this is how you *want* it to run?

Reply:
`n` and `m` are the same as before.

I've removed the `FIX THIS` comment.  It was old.

And, there is a new method to read the JSON files.  In `zdm/io.py`